src/lib/transaction_snark/test/zkapp_preconditions/zkapp_preconditions.ml

```diff
@@ -993,4 +1011,11 @@ let%test_module "Account precondition tests" =
                   failwith
                     "Expected transaction to fail due to invalid account \
                      precondition in the fee payer" ) )
+
+    let () =
+      match Sys.getenv_opt "PROOF_CACHE_OUT" with
```

Comment:
What happens here if we run unit tests with `-only-test` flags? Would we get a proof cache out at the end which is missing some of the necessary proofs?

Reply:
Only if you also set the `PROOF_CACHE_OUT` environment variable (see [here](https://github.com/MinaProtocol/mina/pull/14031/files#diff-e4944a1c552b2b345674ed2c322f75eac44e78cc0328ada6fe603d5098084f0a)).

Reply:
So basically: don't do that :)